software/yarr/egg-yarr.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-10-17T22:24:46-04:00",
+    "name": "Yarr",
+    "author": "sotoluis90@gmail.com",
+    "description": "Yarr is a selfhosted feed aggregator",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:debian": "ghcr.io\/parkervcp\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": "echo \"{{USERNAME}}:{{PASSWORD}} > .\/auth.conf && .\/yarr -addr 0.0.0.0:{{SERVER_PORT}} -auth-file=.\/auth.conf -db=.\/feed.sql",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"starting server\",\r\n    \"userInteraction\": []\r\n}",
```

Comment:
userInteraction is not more used

Reply:
is it safe to just remove that line or whats your actual suggestion here? i'm not sure how to resolve this

Reply:
Just remove that line. And remove the, at the end of the line before